pkg/controller/migmigration/migrate.go

```diff
@@ -90,11 +90,20 @@ func (r *ReconcileMigMigration) migrate(migration *migapi.MigMigration) (int, er
 		return 0, err
 	}
 
+	// TODO: SYNC WITH JEFF TO GET OPINION ON THIS HACK
```

Comment:
This is the part I think we might want to rework. Basically we are waiting until we know the copy has been completed and we set an annotation that is used to know if we can skip creating stage pods the next go round. I think that setting a temp var at the beginning of `task.Run()` will give us the same ability. Curious what you think @jortel.

Reply:
Seems like the consensus is to use the phase assuming the following changes:
- Rewind would be prevented.  It will always reflect the _highest_ phase completed.
- The phase would support order comparisons like phase _After()_ or _>_, _<_.